public/css/contactUs.css

```diff
@@ -40,6 +40,11 @@ input{
 aside {
 	color: white;
 }
+
+.p-e {
+    width: vw;
```

Comment:
What does this do?

Reply:
That was when I was trying to fix the page that webdot later did. It actually does nothing because his changes have removed that class